### ISSUE TITLE
/cloud/storage Ceph logo not loading [ubuntu/chrome]

### ISSUE THREAD

**Issue by deadlight** [canonical/ubuntu.com#169]
## Context

The ceph logo is 404ing and the alt text is incorrectly saying "Swift logo"
## Process
- Navigate to the page and behold the broken image
## Expected result

The image should work and, if it doesn't, the alt text should be correct.
## Current result

The image doesn't load and the alt text is incorrect
